components/catalogs/front/src/components/ProductSelection/criteria/AttributeNumberCriterion/AttributeNumberCriterion.tsx

```diff
@@ -0,0 +1,81 @@
+import React, {FC} from 'react';
+import {CloseIcon, IconButton, List} from 'akeneo-design-system';
+import {CriterionModule} from '../../models/Criterion';
+import {AttributeNumberCriterionState} from './types';
+import {useAttribute} from '../../hooks/useAttribute';
+import {ErrorHelpers} from '../../components/ErrorHelpers';
+import {AttributeNumberOperatorInput} from './AttributeNumberOperatorInput';
+import {AttributeNumberValueInput} from './AttributeNumberValueInput';
+import {ScopeInput} from '../../components/ScopeInput';
+import {LocaleInput} from '../../components/LocaleInput';
+import {CriterionField, CriterionFields} from '../../components/CriterionFields';
+import {Operator} from '../../models/Operator';
+import {useTranslate} from '@akeneo-pim-community/shared';
+
+const AttributeNumberCriterion: FC<CriterionModule<AttributeNumberCriterionState>> = ({
+    state,
+    errors,
+    onChange,
+    onRemove,
+}) => {
+    const translate = useTranslate();
+    const {data: attribute} = useAttribute(state.field);
+    const hasError = Object.values(errors).filter(n => n).length > 0;
+    const showValueInput = [
+        Operator.EQUALS,
+        Operator.NOT_EQUAL,
+        Operator.LOWER_THAN,
+        Operator.LOWER_OR_EQUAL_THAN,
+        Operator.GREATER_THAN,
+        Operator.GREATER_OR_EQUAL_THAN,
+    ].includes(state.operator);
```

Comment:
Optional but I think it's clearer when the value is hidden
```suggestion typescript
    const showValueInput = ![
        Operator.IS_EMPTY,
        Operator.IS_NOT_EMPTY,
    ].includes(state.operator);
```

Reply:
You are probably right, but we already did like this in the others, so I'm going to leave it as it is.